packages/kubernetes-1.20/credential-provider-config.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: kubelet.config.k8s.io/v1beta1
+kind: CredentialProviderConfig
+{{! This isn't right as the condition is more than just whether `providers` is "truey" but whether any are enabled }}
```

Comment:
If kubelet tolerates an empty `providers` list in the YAML then this is probably "right enough" if not perfect.